src/main/resources/templates/thymeleaf/application/application-statistics.html

```diff
@@ -50,11 +50,21 @@ <h1 th:text="#{applications.statistics}">Abwesenheitsstatistik</h1>
               id="statistics-csv-download-link"
               th:href="@{/web/application/statistics/download (from=${from}, to=${to}, page=${statisticsPagination.page.number + 1}, size=${statisticsPagination.page.size}, sort=${sortQuery}, query=${query})}"
               class="icon-link tw-px-1"
-              th:data-title="#{action.download}"
+              th:data-title="#{applications.statistics.button.hover}"
               download
             >
-              <svg th:replace="icon/download::svg(className='tw-w-5 tw-h-5')"></svg>
-              <span class="tw-sr-only" th:text="#{action.download}"></span>
+              <svg th:replace="icon/download::svg(className='tw-w-5 tw-h-5')"></svg>&nbsp;
+              <span th:text="#{applications.statistics.button}"></span>
+            </a>
+            <a
+              id="export-csv-download-link"
```

Comment:
Wenn ein Element das `id` Attribut hat, dann muss das irgendwo verwendet werden. (z. B. Volltextsuche auf dem Projekt. Dann findet man den kopierten `statistics-csv-download-link`)

Verwendung kann sein
* referenziert von einem Link (`<a href="#awesome-id">`)
* referenziert von `aria` Attributen
* referenziert in JavaScript irgendwo
* referenziert von einem `Turbo-Stream` damit das Element bei Clientseitiger Aktion aktualisiert wird

Bei uns ist letzteres interessant. Sonst wird die URL nicht angepasst, wenn die Pagination geändert wird. (mit aktiviertem JavaScript)